test/Streamly/Test/Data/Array/Foreign.hs

```diff
@@ -123,13 +140,15 @@ testStripNull = do
 
 -- Instead of hard coding 10000 here we can have maxStreamLength for operations
 -- that use stream of arrays.
+{-
 concatArrayW8 :: Property
 concatArrayW8 =
     forAll (vectorOf 10000 (arbitrary :: Gen Word8))
         $ \w8List -> do
               let w8ArrList = A.fromList . (: []) <$> w8List
               f2 <- S.toList $ AS.concat $ S.fromList w8ArrList
               w8List `shouldBe` f2
+              -}
```

Comment:
Instead of disabling the test, can we move this to a separate Array.Stream.Foreign module corresponding to the Streamly.Internal.Data.Array.Stream.Foreign?